base/siteorigin-widget.class.php

```diff
@@ -203,7 +203,12 @@ function generate_and_enqueue_instance_styles( $instance ) {
 					}
 					else {
 						// Fall back to using inline CSS if we can't find the cached CSS file.
-						siteorigin_widget_add_inline_css( $this->get_instance_css( $instance ) );
+						// Try get the cached value.
+						$css = wp_cache_get( $hash, 'siteorigin_widgets' );
```

Comment:
Although the hash here is pretty much always going to be unique, it might be a good idea to include `$this->id_base` of the widget as the cache key. It just feels a little neater, because technically all widgets share the same hash space, so could clash.

We're using the hash this way in a few places:

https://github.com/siteorigin/so-widgets-bundle/blob/develop/base/siteorigin-widget.class.php#L901
https://github.com/siteorigin/so-widgets-bundle/blob/develop/base/siteorigin-widget.class.php#L182